Add openwrt

lyj0309 opened the pull request.

lyj0309 committed:
Add openwrt

hasecilu reviewed (changes requested):
Hello, I seems like a good addition to me despite not having some kind of filetype, it's a relevant project.

Regarding the SVG, could you upload it as a single object (merge all paths into a single one) with black color in a square canvas.

lyj0309 committed:
Update svg

lyj0309 commented:
ok, i updated svg as your request

hasecilu reviewed (approved):
Changes are good

hasecilu commented:
Hi @Finii, I think we could make a release before NF in order to include this logo, what do you think? For the moment I don't have ant other logo planned though.

Finii commented:
@hasecilu Happy new year ;-)

Sure! Not sure about the NF release date though. There's not much yet (I believe :thinking: ) apart from the Devicons that I would not have merged here in their form (quality wise). Ended up now to hand-inspect all 500 icons :-(
Any opinion on the NF schedule? :grinning:

hasecilu commented:
Hey, happy new year! :tada: 

I think there is no hurry to release soon, I just wanted to make sure that any changes made here arrive to next version. I have seen that people doesn't seem to update right away and even some weeks can pass.

Finii commented:
The svg
* has a lot cruft in it (i.e. unneeded comments and properties)
* it uses `transform` which sometimes confuses `fontforge`, recommend applying it instead of describing.
* I'm not sure the clip path is interpreted by fontforge at all, I would work that into the real path instead.
  See no reason why such a simple outline would need a clipbath at all :thinking: That must be some manual cleanup leftover?

![image](https://github.com/user-attachments/assets/e0d95e64-d9e5-416c-af7c-1deba10f4b11)

![image](https://github.com/user-attachments/assets/0842e31e-9b26-4182-aaa3-0fdb0c38a29a)

Finii committed:
openwrt: Remove transform

and scale down to 300x300
and drop all metadata

Signed-off-by: Fini Jastrow <ulf.fini.jastrow@desy.de>

Finii committed:
Correct icons.tsv

The OpenWrt icon is added to the TSV with blanks instead of tabs.
It's a tsv: Tab Separated Values

Signed-off-by: Fini Jastrow <ulf.fini.jastrow@desy.de>
